post_installation_configuration/machine-configuration-tasks.adoc

```diff
@@ -12,6 +12,13 @@ Aside from a few specialized features, most changes to operating systems on {pro
 
 Tasks in this section describe how to use features of the Machine Config Operator to configure operating system features on {product-title} nodes.
 
+[IMPORTANT]
+====
+NetworkManager stores new network configurations to /etc/NetworkManager/system-connections/ in a key-file format
+
+Previously, NetworkManager stored new network configurations to /etc/sysconfig/network-scripts/ in the ifcfg format. Starting with RHEL 9.0, RHEL stores new network configurations at /etc/NetworkManager/system-connections/ in a key-file format. The connections for which the configurations are stored to /etc/sysconfig/network-scripts/ in the old format still work uninterrupted. Modifications in existing profiles continue updating the older files.
```

Comment:
```suggestion
Previously, NetworkManager stored new network configurations to /etc/sysconfig/network-scripts/ in the ifcfg format. Starting with RHEL 9.0, RHEL stores new network configurations at /etc/NetworkManager/system-connections/ in a key-file format. The connections configurations stored to /etc/sysconfig/network-scripts/ in the old format still work uninterrupted. Modifications in existing profiles continue updating the older files.
```
That one sentence confused me a bit. wdyt of this edit?

Reply:
Ack.